entities/src/main/java/com/flipkart/varadhi/entities/GetMessagesRequest.java

```diff
@@ -0,0 +1,35 @@
+package com.flipkart.varadhi.entities;
+
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class GetMessagesRequest {
```

Comment:
I think we should put all entities specific to web api in a web package. preferably versioned? as in "web.v1."?

Reply:
aligned, will take independent of this PR. There are other entities as well, which would need to be moved.